src/cr/cube/cube.py

```diff
@@ -47,29 +47,48 @@ def __init__(self, cube_responses, transforms, population, min_base):
     @lazyproperty
     def can_show_pairwise(self):
         """True if all 2D cubes in a multi-cube set can provide pairwise comparison."""
-        if len(self._cubes) < 2:
+        if len(self.cubes) < 2:
             return False
 
         return all(
             all(dt in DT.ALLOWED_PAIRWISE_TYPES for dt in cube.dimension_types[-2:])
             and cube.ndim >= 2
-            for cube in self._cubes[1:]
+            for cube in self.cubes[1:]
         )
 
     @lazyproperty
-    def description(self):
-        """str description of first cube in this set."""
-        return self._cubes[0].description
+    def cubes(self):
+        """Sequence of Cube objects containing data for this analysis."""
+
+        def iter_cubes():
+            """Generate a Cube object for each of cube_responses.
+
+            0D cube-responses and 1D second-and-later cubes are "inflated" to add their
+            missing row dimension.
+            """
+            for idx, cube_response in enumerate(self._cube_responses):
+                cube = Cube(
+                    cube_response,
+                    cube_idx=idx if self._is_multi_cube else None,
+                    transforms=self._transforms_dicts[idx],
+                    population=self._population,
+                    mask_size=self._min_base,
+                )
+                # --- numeric-measures cubes require inflation to restore their
+                # --- rows-dimension, others don't
+                yield cube.inflate() if self._is_numeric_measure else cube
+
+        return tuple(iter_cubes())
```

Comment:
Why does `.cubes` need to be public now?

Reply:
i need to check for the first cube of the cubeset in the exporter. So the options are 1. make this public, 2. make CubeSet subscriptable. I chose this way :D

Reply:
What specifically do you mean by "... check for the first cube of the cubeset ..."? You need to check that it's there? Or you need to inspect some attributes for some reason? Please describe more fully so I can understand. There are a number of cases where work is done in exporter that should be done here, so that's what I'm looking to understand.

Reply:
ok reverted